src/pages/MutatePolicyPage/Fields/FieldsAmbities/FieldsAmbities.tsx

```diff
@@ -0,0 +1,74 @@
+import {
+    FieldLabel,
+    FormikDate,
+    FormikInput,
+    FormikTextArea,
+} from '@pzh-ui/components'
+
+import { ContainerFormSection } from '@/components/Container'
+import { FormFieldInputContainer } from '@/components/Form'
+
+export interface FieldsAmbitiesProps {}
+
+const FieldsAmbities = ({}: FieldsAmbitiesProps) => (
+    <>
+        <ContainerFormSection
+            titel="Algemene informatie"
+            beschrijving="De algemene informatie bevat een duidelijke titel.">
+            <FormFieldInputContainer>
+                <FieldLabel
+                    name="Titel"
+                    label="Titel"
+                    description="Formuleer in enkele woorden de titel van deze ambitie."
+                />
+                <FormikInput optimized={true} name="Titel" type="text" />
```

Comment:
@Aidenbuis You can provide a label and description property to the Formik elements and it will add the FieldLabel above the field. So you don't need to manually add it yourself. And optimized is true by default, so you can remove that.

Reply:
Awesome!